subprojects/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftIncrementalBuildIntegrationTest.groovy

```diff
@@ -33,11 +32,12 @@ import org.gradle.nativeplatform.fixtures.app.SourceElement
 import org.gradle.nativeplatform.fixtures.app.SwiftApp
 import org.gradle.nativeplatform.fixtures.app.SwiftLib
 import org.gradle.test.fixtures.file.TestFile
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.UnitTestPreconditions.NotMacOs
 import org.gradle.util.internal.VersionNumber
-import spock.lang.IgnoreIf
 
 // See https://github.com/gradle/dev-infrastructure/issues/538
-@IgnoreIf({ OperatingSystem.current().isMacOsX() })
+@Requires(NotMacOs)
```

Comment:
I'd seriously consider adding a `not` option to `@Requires`, e.g., `@Requires(not=MacOs)`. Instead of having to create all these not variants.